crates/mipsy_web/src/pages/main/app.rs

```diff
@@ -89,26 +89,23 @@ pub fn render_app() -> Html {
                             // if window element is on the page, create, leak, and add the onchange callback
                             // only if we have not already added it
                             if window().unwrap().get("editor").is_some()
-                                && !*on_content_change_closure_handle
                             {
                                 let cb = Closure::wrap(Box::new(move || {
                                     let editor_contents = crate::get_editor_value();
+                                    let editor_contents2 = editor_contents.clone();
+                                    file3.set(Some(editor_contents2));
```

Comment:
this is the main fix, realized I can save to in-program memory on any change in the editor